website/addons/googledrive/tests/test_models.py

```diff
@@ -7,22 +7,22 @@
 from dateutil import relativedelta
 
 from framework.auth import Auth
+from framework.exceptions import PermissionsError
 from tests.base import OsfTestCase
-from tests.factories import UserFactory, ProjectFactory
+from tests.factories import UserFactory, ProjectFactory, ExternalAccountFactory
 from website.addons.base import exceptions
 
 from website.addons.googledrive.client import GoogleAuthClient
 from website.addons.googledrive.model import (
     GoogleDriveUserSettings,
     GoogleDriveNodeSettings,
-    GoogleDriveOAuthSettings,
     GoogleDriveGuidFile,
 )
 from website.addons.googledrive.tests.factories import (
     GoogleDriveNodeSettingsFactory,
     GoogleDriveUserSettingsFactory,
-    GoogleDriveOAuthSettingsFactory,
 )
+from website.addons.googledrive import model
```

Comment:
Reorganize and consolidate imports. In particular: `from website.addons.googledrive import model` and `from website.addons.googledrive.model import` do not both need to happen. Several of the individual imports are reused and/or redundant.